files/build_templates/buffers_config.j2

```diff
@@ -172,4 +177,16 @@ def
 {% endfor %}
     }
 {% endif %}
+,
```

Comment:
Indent `,`?

Reply:
Or better place it in above both `if` and `else` block.

---
In reply to: [541480018](https://github.com/Azure/sonic-buildimage/pull/6194#discussion_r541480018) [](ancestors = 541480018)

Reply:
Let's add an indent.
To move it into the `if` block may introduce more complexity because
- the case the comma needs to be added to both `if` and `else` branch
- in the `if` branch, the case  `defs.generate_queue_buffers(port_names_active)` not defined should also been handled, which makes the code complex and hard to understand.